ntoskrnl/ntoskrnl.spec

```diff
@@ -930,6 +930,16 @@
 @ stdcall PfxRemovePrefix(ptr ptr)
 @ stdcall PoCallDriver(ptr ptr)
 @ stdcall PoCancelDeviceNotify(ptr)
+@ stdcall -version=0x600+ PoFxActivateComponent(ptr long long)
+@ stdcall -version=0x600+ PoFxCompleteDevicePowerNotRequired(ptr)
+@ stdcall -version=0x600+ PoFxCompleteIdleCondition(ptr long)
+@ stdcall -version=0x600+ PoFxCompleteIdleState(ptr long)
+@ stdcall -version=0x600+ PoFxIdleComponent(ptr ptr long)
+@ stdcall -version=0x600+ PoFxRegisterDevice(ptr ptr ptr)
+@ stdcall -version=0x600+ PoFxReportDevicePoweredOn(ptr)
+@ stdcall -version=0x600+ PoFxSetDeviceIdleTimeout(ptr int64)
+@ stdcall -version=0x600+ PoFxStartDevicePowerManagement(ptr)
+@ stdcall -version=0x600+ PoFxUnregisterDevice(ptr)
```

Comment:
Since you linked ntoskrnl together with ntoskrnl_vista anyway, might it be worth to export all other functions those are in ntoskrnl_vista here? This is probably outside of the context of this PR, and is not required by these changes, but nevertheless it needs to be done, to have these functions properly used for NT6 and later.
Also, these PoFx* specific functions are actually 0x602+ (Windows 8+). So, since you raised the target version for ntoskrnl_vista to 0x602 anyway, it probably needs to export them for 0x602+ version instead (in case it compiles correctly of course).